locales/en/messages.json

```diff
@@ -1045,7 +1045,7 @@
         "message": "&lt;Select One&gt;"
     },
     "featureRX_PPM": {
-        "message": "PPM RX input"
+        "message": "PPM (single wire PWM)"
```

Comment:
This is incorrect. This protocol uses PPM, not PWM.

Reply:
Thanks, yes.  PPM is a series of pulses and their offset from the clock provides a PWM equivalent, so many people think of it as 'serial PWM', though it isn't, strictly.  I was trying to differentiate it from PWM.
I wonder if the list order should be changed to put serial at the top and PPM above CPPM?
Those old forms are archaic these days.